src/main/java/dp/handler/Handler.java

```diff
@@ -122,7 +122,13 @@ public void listen(final ExportedFile message) {
         MessageType messageType = GetMessageType(message);
 
         try {
-
+            if (message.getRowCount() != null){
```

Comment:
I think it would be good to set message.getRowCount() as a variable as it is used more than once.
```suggestion
int rowCount = message.getRowCount()
            if (rowCount != null){
```